.github/workflows/approve-snapshots.yml

```diff
@@ -22,22 +22,25 @@ jobs:
         with:
           ref: ${{ steps.getBranchName.outputs.branch }}
 
-      - name: Install pnpm ⚙️
-        uses: pnpm/action-setup@v2
-        with:
-          version: 7.x
-
       - name: Use Node 16 🕹️
         uses: actions/setup-node@v3
         with:
           node-version-file: 'package.json'
-          cache: 'pnpm'
 
-      - name: Cache Cypress binary 📌
+      - name: Install pnpm ⚙️
+        uses: pnpm/action-setup@v2
+        with:
+          version: 7.x
+
+      - name: Restore cache 📌
         uses: actions/cache@v3
         with:
-          path: ~/.cache/Cypress
-          key: cypress-${{ runner.os }}-${{ hashFiles('pnpm-lock.yaml') }}
+          path: |
+            ~/setup-pnpm/node_modules/.bin/store
```

Comment:
The path is hardcoded for now, as I don't want to add an [extra step](https://github.com/pnpm/action-setup#use-cache-to-reduce-installation-time) just to run `pnpm store path`. I've suggested storing the path as an output of the `pnpm/action-setup` step in https://github.com/pnpm/action-setup/issues/68#issuecomment-1346181326. To be continued.

Reply:
Yeah, it is fine like this, no worries.